### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by HypnoMama**
-window scroll to top on failed submission
-refactor map fade to remove jQuery library
-fix pagination showing too many pages
-fix map zoom for opportunity submission
-remove portal header from all pages
-add terms of use
-add price to opportunity schema
-add price inputs for sale or rental amounts
-refactor to handle multiple pids/parcels
-map shows multiple polygons
-fix radio story
-add terms tests and stories

### CODE REVIEW

File: cit3.0-web/src/components/ChangeView/ChangeView.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useMap } from "react-leaflet";
 
 export default function ChangeView({ center, zoom }) {
-  if (center && zoom) {
+  if (center[0] !== 54.1722) {
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/ChangeView/ChangeView.js:4]: Just for my own understanding, what does this fix?

File: cit3.0-web/src/components/FormComponents/Radios.stories.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
-import React from "react";
-import { action } from "@storybook/addon-actions";
+import React, { useState } from "react";
+// import { action } from "@storybook/addon-actions";
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/FormComponents/Radios.stories.js:2]:
```suggestion

```

File: cit3.0-web/src/components/Map/Map.js [hunk 3]
```diff
@@ -133,7 +130,7 @@ export default function Map({
           : null}
       </>
     );
-    zoomLevel = 2;
+    zoomLevel = 5; // do not change!
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/Map/Map.js:133]: How come this is this kept at 5?

**Review by BrendanBeachBC — APPROVED** [review]
I have a couple questions just for my own understanding, but looks good to me!

File: cit3.0-web/src/components/ChangeView/ChangeView.js (hunk 1 above)
Comment (HypnoMama) [reply cit3.0-web/src/components/ChangeView/ChangeView.js:4]: This prevents zooming in on the initial render.

File: cit3.0-web/src/components/Map/Map.js (hunk 3 above)
Comment (HypnoMama) [reply cit3.0-web/src/components/Map/Map.js:133]: 5 gives us an initial view of all of BC, 2 gives us all of North and South America :-)

**Pull request merged**